frontend/components/header.js

```diff
@@ -16,18 +16,39 @@ class Header extends HTMLElement {
   connectedCallback() {
     const rawUsername = localStorage.getItem('signedIn') === "true" ? localStorage.getItem('username') : "there";
     const username = escapeHTML(rawUsername);
-
+    const profilePicture = username === 'there' ? `assets/images/default-avatar.jpeg` : `https://github.com/${username}.png`;
```

Comment:
May we plz create a new variable called `const isSignIn` and use that instead of checking twice